bedrock/products/templates/products/vpn/resource-center/includes/vpn_article_card.html

```diff
@@ -0,0 +1,16 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from 'macros-protocol.html' import card %}
+
+{{ card (
+  title=article.info.title,
+  ga_title=article.info.title,
+  aspect_ratio='mzp-has-aspect-16-9',
+  desc=article.info.blurb,
```

Comment:
It will exist if we get this far in the rendering step -- `info` would only be missing if there is no Contenful data in the DB that matches the relevant slug, and the view catches this. Without the catch in the view, it will 500 if somoene enters a random slug

Reply:
@stevejalim do you know this? I'm not sure if it always exists

Reply:
Thanks for the context!